src/components/Board/PanZoomRotate.js

```diff
@@ -50,13 +51,25 @@ const PanZoomRotate = ({ children }) => {
   }, [config, setDim]);
 
   const onWheel = (e) => {
-    if (e.altKey || e.ctrlKey) {
+    if (e.altKey) {
```

Comment:
Can you keep ignoring ctrlKey for other os to avoid navigator zoom default action pls.